test/verify/check-metrics

```diff
@@ -201,6 +207,8 @@ class TestHistoryMetrics(MachineCase):
         b = self.browser
         m = self.machine
 
+        b.wait_timeout(60)
```

Comment:
Oh, does this work just like that without `with`?

Reply:
Yes, `with` just restores the timeout at the end of the context. Like this it's effectively equivalent to `b.cdp.timeout = 60`, but looks a bit less hacky/rude.